pkg/metrics/metrics.go

```diff
@@ -101,6 +115,21 @@ func (mm *MetricsManager) RecordOperationErrorMetrics(
 	klog.Infof("Recorded PDCSI operation error code: %q", errCode)
 }
 
+func (mm *MetricsManager) RecordMountErrorMetric(fs_format string, err error) {
```

Comment:
Can we have some unit tests for this function?

In the past errors.As had some weird bugs of not being able to parse the error type as we expected. 

Other than this, LGTM.

Reply:
Got it, done.